coeus-impl/src/main/java/org/kuali/coeus/sys/framework/workflow/KcPostProcessor.java

```diff
@@ -31,4 +33,15 @@ public ProcessDocReport doRouteStatusChange(DocumentRouteStatusChange statusChan
         return ((PostProcessorService) KcServiceLocator.getService("kcPostProcessorService")).doRouteStatusChange(statusChangeEvent);
     }
 
+    @Override
+    public ProcessDocReport doRouteLevelChange(final DocumentRouteLevelChange levelChangeEvent) throws Exception {
+        return ((PostProcessorService) KcServiceLocator.getService("kcPostProcessorService")).doRouteLevelChange(levelChangeEvent);
```

Comment:
I think we should do the following:

1) make the KcPostProcessor implement the PostProcessor interface NOT extend the KualiPostProcessor
2) make all methods defer directly to the KcPostProcessorService

Without doing this some methods are using the KcPosProcessorService and some are using the KualiPostProcessorService.  This is confusing and not ideal from a spring customization standpoint.